src/data/grammar.json

```diff
@@ -40,15 +41,18 @@
         "The Importance of [_thing:#card#,#cardCategory.s#,#creatureTypes.s#,#gameConcept#]#_thing# in [_where:#format#,#recentCardSet#]#_where#",
         "#pros# Don't realize that #tech.s# #verbDefeat# #obstacle.capitalize#",
         "The Secret to #improvementGoal.capitalizeAll# is #deckChange.capitalize#",
-        "In today's Magic Story: #character#'s #adj# history with #character# finally revealed"
+        "In today's Magic Story: #character#'s #adj# history with #character# finally revealed",
+        "Video: MTG – How I would improve #thing# in Magic: the Gathering, Part #num2-9#",
```

Comment:
I guess they're using it for indexing search results? If we were linking a video then it might be clearer that it's just the title of the video being repeated. As it is it still seems redudant.

And, I don't understand what you're saying about using real names..what's bad about this headline vs. the other ones? The suggestion is primarily because since it's in the first person, it reads as if the opinion is coming from the twitter account, rather than being the title of something else that's being reposted.